src/readinglistbar.cpp

```diff
@@ -122,6 +124,20 @@ void ReadingListBar::onExport()
         app->showMessage(gt("export-reading-list-error"), gt("error-title"), QMessageBox::Information);
 }
 
+void ReadingListBar::onImport()
+{
+    auto app = KiwixApp::instance();
+    auto library = app->getLibrary();
+    QString fileName = QFileDialog::getOpenFileName(app->getMainWindow(),
+                                                    gt("save-file-as-window-title"),
```

Comment:
"Save file as..." is a misleading window title when opening a file

Reply:
Done